pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci</groupId>
         <artifactId>jenkins</artifactId>
-        <version>1.76</version>
+        <version>1.88</version>
```

Comment:
This switched the bytecode format to Java 11, unfortunately, breaking e.g. https://github.com/jenkinsci/mercurial-plugin/pull/235. Needs to be overridden.

Also while I am here, there is something wrong with the POM that breaks Dependabot’s changelog display. I think I need to override `url`.